server/test/controllers/admin/ProgramAdminManagementControllerTest.java

```diff
@@ -163,4 +163,65 @@ public void update_programNotFound_returnsNotFound() {
 
     assertThat(result.status()).isEqualTo(NOT_FOUND);
   }
+
+  @Test
+  public void add_succeeds() {
```

Comment:
Can you test the error case?

Reply:
I'm not sure how to test this. I tried writing one for the case where `makeProgramAdmins()` returns `Optional.empty`, but this ends up just testing that the return code is `OK` instead of `SEE_OTHER`. What do you think?

Reply:
I looked through our tests to see if there is a way of asserting something about the `Toast` but did not find anything. I certainly may have missed something, though.

Reply:
I think you can look at the returned html in the response, but maybe not necessary. Fine to leave it as is in that case, thanks for checking!